devops/azure-pipelines.yml

```diff
@@ -0,0 +1,91 @@
+# This pipeline does not implement the analysis template from the 
+# project-registry-devops repository because there was an issue with
+# analysis of an Angular build when checking out multiple repos.
+
+# This pipeline will trigger on commits or pull-requests made to the branch-dev
+# or main branches.  Note that static analysis with SonarCloud is only performed
+# on "long-lived" branches, which are designated as 'main' and any branch name
+# prefixed with 'branch-'.
```

Comment:
that's true, but sonarcloud will also do PR analyses. this won't be the "full analysis", but it's pretty much what you want on a PR.

Reply:
clarify the comment so it's not longer misleading. we are getting an analysis, if not the full analysis, on non-long-lived branches, as long as there's a pull request from them that activates our pr trigger.